src/devtoolsPanel.ts

```diff
@@ -57,6 +57,7 @@ export class DevToolsPanel {
         this.panelSocket.on("setState", (msg) => this.onSocketSetState(msg));
         this.panelSocket.on("getUrl", (msg) => this.onSocketGetUrl(msg));
         this.panelSocket.on("openInEditor", (msg) => this.onSocketOpenInEditor(msg));
+        this.panelSocket.on("close", () => this.onSocketClose());

```

Comment:
You are failing a test here because it used to make sure that we were handling all the events, but now you've added a new one it doesn't know about.

Be sure to update that test, and then add a new one which checks that firing 'close' will actually dispose the panel.